tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,34 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+import time
+from gaiatest.apps.bzlite.app import BugzillaLiteStage
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_app = {
```

Comment:
Many lines are related to the installation of the app. I'd put these lines in a function, so the setup() is at the same level of details:
``` python
def setUp(self):
    GaiaTestCase.setUp(self)
    self.connect_to_local_area_network() # <= No details about how to connect
    self.install_app({
        'name': 'Bugzilla Lite Stage',
        'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'
    }) # <= No details about how to install
    self.push_resource('IMG_0001.jpg') # <= No details about how to send a file
```